fsspec/fuse.py

```diff
@@ -141,20 +167,43 @@ def run(fs, path, mount_point, foreground=True, threads=False):
         character.
     mount_point: str
         An empty directory on the local file-system where the contents of
-        the remote path will appear
+        the remote path will appear.
     foreground: bool
         Whether or not calling this function will block. Operation will
         typically be more stable if True.
     threads: bool
         Whether or not to create threads when responding to file operations
         within the mounter directory. Operation will typically be more
         stable if False.
+    log_filename: str
+        The FUSE log file name. If not provided, the logging feature is
```

Comment:
Presumably, this function could be called from code which sets up logging in some other custom way. Possibly this should be done in the app main function rather than here.

Reply:
Agreed.

Reply:
OK.